components/header/header.js

```diff
@@ -0,0 +1,72 @@
+import "../link-button/link-button.js";
+
+const template = document.createElement("template");
+template.innerHTML = `
+  <style>
+    :host {
+      position: fixed;
+      top: 0;
+      left: 0;
+      padding: 20px 0;
+      width: 100%;
+      border-bottom: 1px solid #D3D3D3;
+      background-color: rgba(255, 255, 255, 0.8);
+    }
+
+    header {
+      width: 80%;
+      margin: 0 auto;
+      justify-content: space-between;
+      display: flex;
+      align-items: center;
+    }
+
+    .buttons-container {
+      display: flex;
+    }
```

Comment:
So you're saying you'll change the responsive design for the buttons in a future PR?
You can resolve this discussion if there's no issue.

Reply:
Yes I thought i'd be better to sperate commits even in one PR. But not for buttons per se only the layout since the detail styles of button would be handled in link-button right?

Reply:
You're right.
- Layout for buttons should be controlled in the header component.
- detail styles of button itself should be controlled in the link-button component